### PR TITLE
Release

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

File: src/components/models/googleSign.js [hunk 1]
```diff
@@ -182,7 +185,7 @@ const GoogleModel = ({
                         Are you sure you want to share this Project to Google Classroom?
                       </p>
                       */}
-                      <button type="submit">Confirm</button>
+                      {!loading &&<button type="submit">Confirm</button>}
```
Comment (simonebelotti75) [inline src/components/models/googleSign.js:188]: Do we remove the button at all? Not disabled?

File: src/containers/LtiModel/index.js [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+    dispatch(
+      loadMyProjectsltiAction(
+        decodeURIComponent(match.params.lms_url),
+        decodeURIComponent(match.params.lti_client_id)
+      )
+    );
+  }, []);
+
+  useEffect(() => {
+    allProjects.project.projects && setAllproject(allProjects.project.projects);
```
Comment (simonebelotti75) [inline src/containers/LtiModel/index.js:34]: Please do not use this syntax, instead use if condition
I think I mentioned it before

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/components/models/googleSign.js (hunk 1 above)
Comment (muhammadqamar) [reply src/components/models/googleSign.js:188]: this is the requirement

File: src/containers/LtiModel/index.js (hunk 2 above)
Comment (muhammadqamar) [reply src/containers/LtiModel/index.js]: updated

**Review by simonebelotti75 — APPROVED** [review]

**Pull request merged**